supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -1,7 +1,7 @@
 package cromwell.backend.impl.jes
 
 import cromwell.backend.BackendConfigurationDescriptor
-import cromwell.backend.google.pipelines.v1alpha2.PipelinesApiLifecycleActorFactory
+import cromwell.backend.google.pipelines.v2alpha1.PipelinesApiLifecycleActorFactory
```

Comment:
This means people will now by default be using V2 if they don't change their config. Do we want that ?

Reply:
IMO no, but I could see the argument in both directions.

Reply:
Ok, I don't really have a strong argument either way but I can see that we want to keep V1 by default for now.